pkg/client/dtclient/client.go

```diff
@@ -126,6 +126,8 @@ type SettingsClient interface {
 	// ListSchemas returns all schemas that the Dynatrace environment reports
 	ListSchemas() (SchemaList, error)
 
+	Schema(schemaID string) (schema Schema, err error)
```

Comment:
```suggestion
	GetSchema(schemaID string) (Schema, error)
```
If the return values are clear, skip them. Include the name if their intent is not clear

Reply:
`Schema` vs. `GetSchema` -  go style guide explicitly recommends avoiding `get` prefix (https://google.github.io/styleguide/go/best-practices#avoid-repetition, https://google.github.io/styleguide/go/decisions#getters)

This would be fine if we used the get for HTTP GET method, post for POST, etc.

Reply:
> If the function involves performing a complex computation or executing a remote call, a different word like Compute or Fetch can be used in place of Get, to make it clear to a reader that the function call may take time and could block or fail.
Note that this method is not a Getter, so just dropping it would not be in the spirit of the styleguide

I would be ok with FetchSchemas.

Reply:
I'll rename it...